conpot/protocols/http/command_responder.py

```diff
@@ -664,10 +663,14 @@ def do_HEAD(self):
         else:
 
             # try to find a configuration item for this GET request
-            entity_xml = configuration.xpath(
-                '//http/htdocs/node[@name="'
-                + self.path.partition('?')[0] + '"]'
-            )
+            try:
+                entity_xml = configuration.xpath(
+                    '//http/htdocs/node[@name="'
+                    + self.path.partition('?')[0] + '"]'
+                )
+            except:
+                entity_xml = None
+                logger.debug('Malformed HTTP:HEAD URN. Failed to handle <%s>. (Client: %s)', self.path, self.client_address)
```

Comment:
afair `%s` formatting in being phased out as per PEP 3101.  I know it's a nuisance.. but using `.format()` might be better for us :wink: